query/src/storages/fuse/fuse_part.rs

```diff
@@ -13,17 +13,36 @@
 // limitations under the License.
 
 use std::any::Any;
+use std::collections::HashMap;
 use std::sync::Arc;
 
 use common_exception::ErrorCode;
 use common_exception::Result;
 use common_planners::PartInfo;
 use common_planners::PartInfoPtr;
 
+#[derive(serde::Serialize, serde::Deserialize, PartialEq)]
+pub struct ColumnMeta {
```

Comment:
Why not use the struct of `block::ColumnMeta`?

Reply:
They may have different fields in the future.